sretoolbox/container/image.py

```diff
@@ -549,3 +555,22 @@ def __str__(self):
         else:
             url = self.url_tag
         return f'{self.scheme}{url}'
+
+    def __contains__(self, item):
```

Comment:
__contains__ is already implemented https://github.com/app-sre/sretoolbox/blob/master/sretoolbox/container/image.py#L489-L490

it's used to find tags... quite used actually: https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/quay_mirror.py#L181

Reply:
haha, yes... i've just realised it